packages/SketchMorph2-Core.package/MorphicMonet.class/instance/defaultBrushSizeSlider.st

```diff
@@ -1,7 +1,7 @@
 defaults
 defaultBrushSizeSlider
 	| slider |
-	slider := SimpleSliderMorph new
+	slider := M2SimpleSliderMorph new
```

Comment:
Another possibility would be to create a class `M2BrushSlider`, which already contains the correct values for `minVal`, `maxVal` etc., which would be semantically more understandable, because I don't expect more sliders to be added to the menu soon.

Reply:
I'll try this.